bot/cogs/bot.py

```diff
@@ -1,10 +1,14 @@
 # coding=utf-8
-from discord import Embed
+import ast
+import time
+
+from discord import Embed, Message
 from discord.ext.commands import AutoShardedBot, Context, command, group
 
 from dulwich.repo import Repo
 
-from bot.constants import PYTHON_GUILD, VERIFIED_ROLE
+from bot.constants import HELP1_CHANNEL, HELP2_CHANNEL, HELP3_CHANNEL
+from bot.constants import PYTHON_CHANNEL, PYTHON_GUILD, VERIFIED_ROLE
```

Comment:
There's no need to have two `import` statements. You can spread the imports over `2` or `3` lines without repeating `import` statements, by using surrounding parentheses.

Reply:
this I don't agree with. multiple imports are fine.

Reply:
understandable, have a nice day